src/pages/home/home.ts

```diff
@@ -73,6 +74,16 @@ export class HomePage {
       /* imgSrc: TODO 'assets/img/bitpay-card-solid.svg' */
       imgSrc: 'assets/img/icon-bpcard.svg'
     },
+    {
+      name: 'coinbase',
+      title: 'Coinbase',
+      body: 'Deposit and withdrawal with your coinbase account.',
```

Comment:
"Deposit and withdraw with your coinbase account"

Reply:
done